docs/contributor/03-10-hyperscaler-account-pool.md

```diff
@@ -82,3 +84,34 @@ metadata:
     tenant-name: {TENANT_NAME}
     hyperscaler-type: "gcp_cf-sa30"
 ```
+
+## Secret Bindings Selection Rules
+
+<!-- rules Overview -->
+HAP evaluates a set of rules to determine what labels to use when querying secret bindings. Input to the rules consists of an SKR's plan, hyperscaler and region. If evaluated to true the rules modify or add labels used in the secret bindings resource query. There are four possible rules to configure:
+* `hap.platformRegionRule` - if evaluated to true the `_<PLATFOR_REGION>` is appended to the `hyperscaler-type` label when searching, refered to as platform region based search,
+* `hap.clusterRegionRule` - if evaluated to true the `_<HYPERSCALER_REGION>` is appended to the `hyperscaler-type` label when searching, refered to as cluster region based search,
+* `hap.sharedRule` - if evaluated to true the `shared: true` label is used when searching, refered to as shared based search,
+* `hap.euAccessRule` - if evaluated to true the `shared: true` label is used when searching, refered to as euAccess based search.
+The configuration is done by specifying above rules in KEB helm values. 
+
+<!-- rules format -->
+Each rule consists of a semicolon separated list of plans that the rule applies to. Additionally, a plan can be extended with `:region` suffix that makes the rule evaluate to true only if a cluster is provisioned in the specified region. List entries comply with the format `<PLAN_ID_1>:<REGION_ID_1>;<PLAN_ID_2>:<REGION_ID_2>`. Either plan or region (but never both) can be specified as wildcard `*` meaning all plans or regions should apply for specific second value. The following example lists valid and invalid configuration values:
+* `trial` - valid, rule will be evaluated to true for the `trial` plan in all regions,
+* `trial:*` - valid, rule will be evaluated to true for the `trial` plan in all regions,
+* `trial:eu` - valid, rule will be evaluated to true for the `trial` plan in the `eu` region,
+
+* `*:eu` - valid, rule will be evaluated to true for all plans in the `eu` region,
+* `eu:*` - invalid, plan must be specified in the first part of `<PLAN_ID>:<REGION_ID>` pair,
+* `*:*` - invalid, at least one of the values must be specified in the `<PLAN_ID>:<REGION_ID>` pair.
+
+* `*:eu;trial:eu` - valid, rule will be evaluated to true for all plans in the `eu` region and for the `trial` plan in the `eu` region, configuration can be duplicated
+* `trial:eu;trial:gcp` - valid, rule will be evaluated to true for trials plans but only in `eu` and `gcp` regions,
+
+<!-- search construction -->
+![alt text](image.png)
+
+<!-- TODO: rules validation -->
+Rules validation is done during the KEB startup. If the configuration is invalid, KEB will not start and an error message will be displayed in the logs. The constraints used for validation include:
+* Rules format check - all the rules must comply with the format specified above.
+* Plan ?and region? existence check.
```

Comment:
It seems it needs rephrasing.

Reply:
I was wondering if we have simple means to implement region check. I remove `?and region?` from the doc for now.